tools/webpack.config.dev.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+var version = require('./../package.json').version;
+var path = require('path');
+var webpack = require('webpack');
+var autoprefixer = require('autoprefixer');
+
+var rootPath = path.join(__dirname, '..');
+
+function root(p){
+  return path.join(rootPath, p);
+}
+
+var distPath = root('dist');
+
+module.exports = {
+  entry: [
+    'webpack-hot-middleware/client?reload=true',
+    './src/entry'
+  ],
+  output: {
+    path: distPath,
+    publicPath: '/dist',
+    filename: 'bundle.js'
+  },
+  plugins: [
+    new webpack.HotModuleReplacementPlugin(),
+    new webpack.NoErrorsPlugin(),
+    new webpack.DefinePlugin({
+      'process.env.NODE_ENV': JSON.stringify('development'),
+      APP_VERSION: JSON.stringify(version)
```

Comment:
For example if you would ever use any react development tools, aka https://github.com/gaearon/redux-devtools or show any debug information, it would be useful.